src/test/kotlin/org/pac4j/vertx/handler/impl/Functions.kt

```diff
@@ -65,7 +64,7 @@ fun logoutHandler(vertx: io.vertx.core.Vertx): Handler<RoutingContext> {
 }
 
 
-fun getProfileManager(rc: RoutingContext): VertxProfileManager<TestOAuth1Profile> {
```

Comment:
Yep, since I'm the only person who's ever written or modified tests for the vertx-pac4j stuff, I'm choosing to permit either Java or Kotlin tests. I assume that's not a problem (I don't see why it should be).